google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -20,6 +20,9 @@ Code inspections for AppEngine Java code.</description>
     <component>
       <implementation-class>com.google.gct.idea.CloudToolsPluginInitializationComponent</implementation-class>
     </component>
+    <component>
+      <implementation-class>com.google.gct.idea.debugger.CloudDebugProcessStateCollector</implementation-class>
```

Comment:
So application components are just intellij instantiated singletons? I should probably use this more.